aveas_openlabel/attributes/open_drive.py

```diff
@@ -37,6 +37,28 @@ class OpenDrive__RoadId(TextData):
     """Is always 'open_drive/road_id'"""
 
 
+@dataclass
+class OpenDrive__LaneId(TextData):
+    """Lane ID the object is moving on corresponding to the Road ID (stated in OpenDrive__RoadId) in the OpenDRIVE file"""
```

Comment:
Add backticks (`) to OpenDrive__RoadId so that the documentation creates a hyperlink to the class.

Reply:
geändert und zusätzlich angewendet in OpenDrive__LanePosition und